### DISCUSSION TITLE
latest/detection/annotators/

### DISCUSSION THREAD

**Discussion by giscus** [roboflow/supervision#1441] [Docs]
# latest/detection/annotators/

A set of easy-to-use utilities that will come in handy in any computer vision project.

https://supervision.roboflow.com/latest/detection/annotators/

**Comment by bejaouiaziz** [comment]
I feel in love with this work

**Reply by SkalskiP** [reply]
That's what I wanted to hear!

**Comment by AndrinoLabGit** [comment]
How do you set the background of LabelAnnotator to be transparent?
